packages/batch-poster-monitor/index.ts

```diff
@@ -221,23 +219,31 @@ const getBatchPosterLowBalanceAlertMessage = async (
     }
   }
 
-  const balance = await parentChainClient.getBalance({
+  const currentBalance = await parentChainClient.getBalance({
     address: batchPoster,
   })
 
-  const lowBalanceDetected =
-    (childChainInformation.parentChainId === 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ETHEREUM * 1e18)) ||
-    (childChainInformation.parentChainId !== 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ARBITRUM * 1e18))
+  // get the gas used in the last 24 hours
+  let gasUsedInLast24Hours = BigInt(0)
+  for (const log of sequencerInboxLogs) {
+    gasUsedInLast24Hours = (
+      await parentChainClient.getTransactionReceipt({
+        hash: log.transactionHash,
+      })
+    ).gasUsed
+  }
+  const currentGasPrice = await parentChainClient.getGasPrice()
+  const balanceSpentIn24Hours = gasUsedInLast24Hours * currentGasPrice
```

Comment:
I think it's more accurate to multiply in the for loop above. You can multiply `gasUsed` by `effectiveGasPrice`. Otherwise, we'd be assuming the same price for the whole 24 hours.